src/components/plots/plot.ts

```diff
@@ -124,7 +108,7 @@ module Plottable {
     }
 
     protected _setupDatasetNodes(dataset: Dataset) {
-      var drawer = this._key2PlotDatasetKey.get(this._keyForDataset(dataset)).drawer;
+      var drawer = this._datasetToDrawer.get(dataset);
```

Comment:
The intention I had for this function was essentially
"Create the DOM nodes in accordance with a drawer" not really
"Create the drawer and the DOM nodes in accordance with it"

I would personally like to keep things simple and go wiht the former.  What do you think?